config/development.js

```diff
@@ -1,4 +1,27 @@
 exports.config = {
   environment: 'development',
+  common: {
+    database: {
+      url: process.env.NODE_API_DB_URL,
+      host: process.env.NODE_API_DB_HOST,
+      port: process.env.NODE_API_DB_PORT,
+      database: process.env.NODE_API_DB_NAME,
+      username: process.env.NODE_API_DB_USERNAME,
+      password: process.env.NODE_API_DB_PASSWORD,
+      schema: process.env.NODE_API_DB_SCHEMA || 'public'
+    },
+    api: {
+      bodySizeLimit: process.env.API_BODY_SIZE_LIMIT,
+      parameterLimit: process.env.API_PARAMETER_LIMIT
+    },
+    session: {
+      header_name: 'authorization',
+      secret: process.env.NODE_API_SESSION_SECRET
+    },
+    rollbar: {
+      accessToken: process.env.ROLLBAR_ACCESS_TOKEN,
+      environment: process.env.ROLLBAR_ENV
+    }
+  },
```

Comment:
You must change this and all the config files because dotenv was added to this repository. Merge with master!